### ISSUE TITLE
Template for issues

### ISSUE THREAD

**Issue by Siponek** [Option7-OctoPuss/MeetYourTesterGame#32]
PREREQUISITES:

What needs to be already done before starting this task

GODOT LOGIC / GODOT UI / FIGMA / GITHUB:

Low-level description of all steps that need to be done

(optional) Triggers

(optional) Examples

BOUNDARIES:

What this tasks handles

What this tasks does not handle

OUTCOMES:

What features are added by this task

**Comment by Siponek** [comment]
In the comments add:

things that will have to be groomed / fixed / defined better
links / resources to help the development
generic suggestions
